lib/puppet/provider/rabbitmq_user/rabbitmqctl.rb

```diff
@@ -73,11 +74,11 @@ def tags=(tags)
   end
 
   def admin
-    if usertags = get_user_tags
-      (:true if usertags.include?('administrator')) || :false
-    else
-      raise Puppet::Error, "Could not match line '#{resource[:name]} (true|false)' from list_users (perhaps you are running on an older version of rabbitmq that does not support admin users?)"
+    if (usertags = get_user_tags)
```

Comment:
@ekohl How about like this?

Reply:
I was thinking about:
```ruby
usertags = get_user_tags
raise ... unless usertags
:true if usertags.include?('administrator') || :false
```

And I'd see if it could handle actual booleans.

Reply:
Look good?